llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1408,47 +1404,45 @@ Value *SpirvLowerGlobal::loadDynamicIndexedMembers(Type *inOutTy, unsigned addrS
         else
           elemLocOffset = BinaryOperator::CreateAdd(locOffset, m_builder->getInt32(stride * idx), "", insertPos);
 
-        auto elem = loadDynamicIndexedMembers(elemTy, addrSpace, indexOperands, operandIdx + 1, elemMeta, elemLocOffset,
+        auto elem = loadDynamicIndexedMembers(elemTy, addrSpace, indexOperands.slice(1), elemMeta, elemLocOffset,
                                               interpLoc, auxInterpValue, insertPos);
         inOutValue = InsertValueInst::Create(inOutValue, elem, {idx}, "", insertPos);
       }
       return inOutValue;
     }
 
-    // for constant index, we only need to load the specified value
-    unsigned elemIdx = cast<ConstantInt>(indexOperands[operandIdx])->getZExtValue();
+    // For constant index, we only need to load the specified value
+    unsigned elemIdx = cast<ConstantInt>(indexOperands[0])->getZExtValue();
     Value *elemLocOffset = nullptr;
     if (isa<ConstantInt>(locOffset))
       elemLocOffset = m_builder->getInt32(cast<ConstantInt>(locOffset)->getZExtValue() + stride * elemIdx);
     else
       elemLocOffset = BinaryOperator::CreateAdd(locOffset, m_builder->getInt32(stride * elemIdx), "", insertPos);
 
-    Value *elem = loadDynamicIndexedMembers(elemTy, addrSpace, indexOperands, operandIdx + 1, elemMeta, elemLocOffset,
+    Value *elem = loadDynamicIndexedMembers(elemTy, addrSpace, indexOperands.slice(1), elemMeta, elemLocOffset,
```

Comment:
nit: I think `.drop_front()` would be a more idiomatic way to pass everything but the first element

Reply:
Thanks you. It is very clear.